x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -762,8 +762,6 @@
     "xpack.lens.xyChart.bottomAxisDisabledHelpText": "この設定は、下の軸が有効であるときにのみ適用されます。",
     "xpack.lens.xyChart.bottomAxisLabel": "下の軸",
     "xpack.lens.xyChart.boundaryError": "下界は上界よりも大きくなければなりません",
-    "xpack.lens.xyChart.chartTypeLabel": "チャートタイプ",
-    "xpack.lens.xyChart.chartTypeLegend": "チャートタイプ",
```

Comment:
so these two belong to the component that was not used but wasn't removed (LayerContextMenu)